default_data.json

```diff
@@ -15721,6 +15721,21 @@
                 "blak111@gmail.com", "kevin.benton@bigswitch.com", "kevinbenton@buttewifi.com", "kevin@benton.pub"
             ]
         },
+        {
+            "launchpad_id": "kevin-carter",
+            "companies": [
+                {
+                    "company_name": "Rackspace",
+                    "end_date": "2019-May-3"
```

Comment:
you got failed schema validation because here must be '03' instead of '3'

Reply:
done.